pytao/util/parsers.py

```diff
@@ -10,194 +21,213 @@ def _parse_str_bool(s):
     '0', 'f', 'F' -> Flase
```

Comment:
Not on this PR but there is a typo `Flase` -> `False` 
```suggestion
    '0', 'f', 'F' -> False
```